cohere_v3_vector/README.md

```diff
@@ -0,0 +1,73 @@
+## Cohere msmarco-v2/embed-english-v3 vector track
```

Comment:
I wonder we should call it msmarco-v2? 
If we want to compare another strategy (BM25 or another model for semantic search) it would be preferable to have a single track. For now it's ok to only test the cohere model but it would be nice to link this track with a dataset rather than a search strategy.